src/components/Settings/SettingsLogs/index.tsx

```diff
@@ -1,14 +1,299 @@
+              <tr className="relative h-24 p-2 text-white">
+                <Table.TD colSpan={4} noPadding>
+                  <div className="flex flex-col items-center justify-center w-screen p-6 lg:w-full">
+                    <span className="text-base">
+                      {intl.formatMessage(messages.noresults)}
+                    </span>
+                    {currentFilter !== 'debug' && (
+                      <div className="mt-4">
+                        <Button
+                          buttonSize="sm"
+                          buttonType="primary"
+                          onClick={() => setCurrentFilter('debug')}
+                        >
+                          {intl.formatMessage(messages.showall)}
+                        </Button>
+                      </div>
+                    )}
+                  </div>
+                </Table.TD>
+              </tr>
+            )}
+            <tr className="bg-gray-700">
+              <Table.TD colSpan={6} noPadding>
+                <nav
+                  className="flex flex-col items-center w-screen px-6 py-3 space-x-4 space-y-3 sm:space-y-0 sm:flex-row lg:w-full"
+                  aria-label="Pagination"
+                >
+                  <div className="hidden lg:flex lg:flex-1">
+                    <p className="text-sm">
+                      {data.results.length > 0 &&
+                        intl.formatMessage(messages.showingresults, {
+                          from: pageIndex * currentPageSize + 1,
+                          to:
+                            data.results.length < currentPageSize
+                              ? pageIndex * currentPageSize +
+                                data.results.length
+                              : (pageIndex + 1) * currentPageSize,
+                          total: data.pageInfo.results,
+                          strong: function strong(msg) {
+                            return <span className="font-medium">{msg}</span>;
+                          },
+                        })}
+                    </p>
+                  </div>
+                  <div className="flex justify-center sm:flex-1 sm:justify-start lg:justify-center">
+                    <span className="items-center -mt-3 text-sm sm:-ml-4 lg:ml-0 sm:mt-0">
+                      {intl.formatMessage(messages.resultsperpage, {
+                        pageSize: (
+                          <select
+                            id="pageSize"
+                            name="pageSize"
+                            onChange={(e) => {
+                              setPageIndex(0);
+                              setCurrentPageSize(Number(e.target.value));
+                            }}
+                            value={currentPageSize}
+                            className="inline short"
+                          >
+                            <option value="5">5</option>
+                            <option value="10">10</option>
```

Comment:
```suggestion
```
I don't think only displaying 5 lines of logs would be very helpful/useful as an option, since this isn't like the requests/users lists where the items take up a lot more vertical space.